packages/playground-examples/copy/ko/TypeScript/Primitives/Any.ts

```diff
@@ -0,0 +1,50 @@
+// Any는 TypeScript의 탈출 구문입니다.
```

Comment:
```suggestion
// any는 TypeScript의 탈출 구문입니다.
```
영문이라서 대문자로 시작한 것 같아요 :)